json/json.mbti

```diff
@@ -12,6 +12,26 @@ pub enum JsonValue {
   Object(@moonbitlang/core/map.Map[String, JsonValue])
 }
 fn JsonValue::debug_write(JsonValue, @moonbitlang/core/builtin.Buffer) -> Unit
+fn JsonValue::get_as_array(JsonValue) -> Option[@moonbitlang/core/vec.Vec[JsonValue]]
+fn JsonValue::get_as_bool(JsonValue) -> Option[Bool]
```

Comment:
the API is too lengthy.
how about
```
inspect(x.array().unwrap()[3].object().unwrap()["xx"])
```

Reply:
How about just remove `get_` but keeping `as_array`?

With quotation mark, we can have: `x.item(3)?.value_as_string("key")?` instead of `x.as_array().unwrap().get(3).unwrap().as_object().unwrap().lookup("key").unwrap().as_string().unwrap()`

Reply:
remove `get_` looks good